Dockerfile

```diff
@@ -14,14 +24,15 @@ RUN curl -o /tmp/dockerize.tgz https://raw.githubusercontent.com/kbase/dockerize
     rm /tmp/dockerize.tgz && \
     adduser condor_pool
 
-RUN cd /root && \
-    curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
-    python get-pip.py && \
-    pip install htcondor  && \
-    rm /root/get-pip.py
+# RUN cd /root && \
+#     curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
+#     python get-pip.py && \
+#     pip install htcondor  && \
```

Comment:
We need the htcondor bindings on the VM that's running condor for check_mk.  It's nice to have them in the container too though.